src/js/player.js

```diff
@@ -2747,22 +2803,48 @@ class Player extends Component {
   }
 
   /**
-   * Get or set the autoplay attribute.
+   * Get or set the autoplay option. When this is a boolean it will
+   * modify the attribute on the tech. When this is a string the attribute on
+   * the tech will be removed and `Player` will handle autoplay on loadstarts.
    *
-   * @param {boolean} [value]
-   *        - true means that we should autoplay
-   *        - false means that we should not autoplay
+   * @param {boolean|string} [value]
+   *        - true: autoplay using the browser behavior
+   *        - false: do not autoplay
+   *        - See `manualAutoplay_` for valid string values
+   *        - *: values other than those listed here or in `manualAutoplay_`
+   *             will be set `autoplay` to true
    *
-   * @return {string}
+   * @return {boolean|string}
    *         The current value of autoplay when getting
    */
   autoplay(value) {
-    if (value !== undefined) {
-      this.techCall_('setAutoplay', value);
+    // getter usage
+    if (value === undefined) {
+      return this.options_.autoplay;
+    }
+
+    let techAutoplay;
+
+    // if the value is a valid string set it to that
+    if (typeof value === 'string' && (/(any|play|muted)/).test(value)) {
       this.options_.autoplay = value;
-      return;
+      techAutoplay = false;
+      // if the value is a boolean set it to that
+    } else if (typeof value === 'boolean') {
+      this.options_.autoplay = value;
+      // any other value sets autoplay to true
+    } else {
+      this.options_.autoplay = true;
+    }
+
+    techAutoplay = techAutoplay || value;
+
+    // if we don't have a tech do not queue up
+    // a setAutoplay call. This option will be passed
+    // in the constructor for the tech
+    if (this.tech_) {
+      this.techCall_('setAutoplay', techAutoplay);
```

Comment:
I think I did this because in the case where autoplay is currently `true` on the tech, we want to set it to `false`, maybe that doesn't remove the attribute like I think that it should, I will test this out as well.